### PR TITLE
Added authorization

### PR DESCRIPTION
**Pull request by chkhanch**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -6,6 +6,18 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
 
-  validates :name, length: { in: 2..30 }
-  validates :surname, length: { in: 2..30 }
+  NAME_LENGTH_RANGE = 2..30
+  SURNAME_LENGTH_RANGE = NAME_LENGTH_RANGE
```
Comment (BohdanChaban) [inline app/models/user.rb]: This constant is not needed. You can use NAME_LENGTH_RANGE for both validations

File: app/models/user.rb [hunk 2]
```diff
@@ -6,6 +6,18 @@ class User < ApplicationRecord
...
+  EMAIL_LENGTH_RANGE = 6..40
+  BOOLEAN_VALUES = [true, false].freeze
```
Comment (BohdanChaban) [inline app/models/user.rb]: This constant is not needed. [true, false] can be used in validation

**Comment by chkhanch** [comment]
Fixed

**Comment by skrix** [comment]
Fix tests for new user functional:
```
Failures:
  1) User is valid with valid attributes
     Failure/Error: expect(user).to be_valid
       expected #<User id: nil, name: "Foo", surname: "Bar", email: "foo@mail.com", created_at: nil, updated_at: nil, role: "student", approved: false> to be valid, but got errors: Role is not included in the list
     # ./spec/models/users_spec.rb:29:in `block (2 levels) in <top (required)>'
  2) User is not valid with invalid email
     Failure/Error: expect(user.errors.messages[:email]).to eq [invalid_error]
     
       expected: ["is invalid"]
            got: ["is invalid", "is too short (minimum is 6 characters)"]
     
       (compared using ==)
     # ./spec/models/users_spec.rb:67:in `block (2 levels) in <top (required)>'
  3) User is not valid without a name
     Failure/Error: expect(user.errors.messages[:name]).to eq [short_error]
     
       expected: ["is too short (minimum is 2 characters)"]
            got: ["can't be blank", "is too short (minimum is 2 characters)"]
     
       (compared using ==)
     # ./spec/models/users_spec.rb:73:in `block (2 levels) in <top (required)>'
  4) User is not valid without a surname
     Failure/Error: expect(user.errors.messages[:surname]).to eq [short_error]
     
       expected: ["is too short (minimum is 2 characters)"]
            got: ["can't be blank", "is too short (minimum is 2 characters)"]
     
       (compared using ==)
     # ./spec/models/users_spec.rb:79:in `block (2 levels) in <top (required)>'
  5) User is not valid without a email
     Failure/Error: expect(user.errors.messages[:email]).to eq [blank_error]
     
       expected: ["can't be blank"]
            got: ["can't be blank", "can't be blank", "is too short (minimum is 6 characters)"]
     
       (compared using ==)
     # ./spec/models/users_spec.rb:85:in `block (2 levels) in <top (required)>'
Finished in 0.23667 seconds (files took 2.91 seconds to load)
```

**Pull request merged**